client/lib/screen-title/index.js

```diff
@@ -19,26 +19,24 @@ const TITLE_UPDATING_ACTIONS = [
 ];
 
 /**
- * Given a Redux store instance, subscribes to the store, updating the page
+ * A middleware that updating the page
```

Comment:
Super minor grammar note, but this should be `Redux middleware that updates the page title` or something.